js/app.jsx

```diff
@@ -517,6 +537,8 @@ var Job = React.createClass({
       ['State', state],
       ['Input', inputs],
       ['Output', cleanJobPath(job.conf.output)],
+      ['Pipe', pipe],
+      ['Step', step],
```

Comment:
If you can give an example of what the values of these fields would be for a job I understand ([this one for example](https://github.com/stripe-internal/zoolander/blob/master/zoolander-jobs/src/main/scala/com/stripe/zoolander/transactionfraud/JoinModelProbShieldData.scala)) I would be happy to opine more :) 

I'm definitely not a Scalding superuser or expert, so take my opinions with a grain of salt (but then again, maybe the non-experts are the ones that need this debugging help the most!).